submitted_models/coro_crystal_sensor_config_1/model.sdf

```diff
@@ -0,0 +1,821 @@
+            </noise>
+          </x>
+          <y>
+            <noise type="gaussian">
+              <mean>0.000000080</mean>
+              <bias_mean>0.000000400</bias_mean>
+            </noise>
+          </y>
+          <z>
+            <noise type="gaussian">
+              <mean>0.000000080</mean>
+              <bias_mean>0.000000400</bias_mean>
+            </noise>
+          </z>
+        </magnetometer>
+      </sensor>
+
+
+      <visual name="d435_dae_front_visual">
+        <pose>0.075 0 -0.03 0 0 0</pose>
+        <geometry>
+	  <mesh>
+	    <scale>0.001 0.001 0.001</scale>
+	    <uri>meshes/realsense_d435.dae</uri>
+	  </mesh>
+        </geometry>
+      </visual>
+	    
+      <sensor name="rgbd_front" type="rgbd_camera">
+        <pose>0.102 0 -0.018 0 0 0</pose>
+        <always_on>1</always_on>
+        <update_rate>30</update_rate>
+        <camera name="rgbd_front">
+          <horizontal_fov>1.50098</horizontal_fov>
+          <lens>
+            <intrinsics>
+              <!-- fx = fy = width / ( 2 * tan (hfov / 2 ) ) -->
+              <fx>343.159</fx>
+              <fy>343.159</fy>
+              <!-- cx = ( width - 1 ) / 2 -->
+              <cx>319.5</cx>
+              <!-- cy = ( height - 1 ) / 2 -->
+              <cy>179.5</cy>
+              <s>0</s>
+            </intrinsics>
+          </lens>
+          <distortion>
+            <k1>0.0</k1>
+            <k2>0.0</k2>
+            <k3>0.0</k3>
+            <p1>0.0</p1>
+            <p2>0.0</p2>
+            <center>0.5 0.5</center>
+          </distortion>
+          <image>
+            <width>640</width>
+            <height>360</height>
+            <format>R8G8B8</format>
+          </image>
+          <clip>
+            <near>0.01</near>
+            <far>300</far>
+          </clip>
+          <depth_camera>
+            <clip>
+              <near>0.1</near>
+              <far>10</far>
+            </clip>
+          </depth_camera>
+          <noise>
+            <type>gaussian</type>
+            <mean>0</mean>
+            <stddev>0.007</stddev>
+          </noise>
+        </camera>
+      </sensor>
+
+
+      <visual name="camera_front_visual">
+        <pose>0.08 0 0.03 0 0 0</pose>
+        <geometry>
+          <box>
+            <size>0.01 0.04 0.04</size>
+          </box>
+        </geometry>
+        <material>
+          <ambient>0.0 0.2 0.0 1.0</ambient>
+          <diffuse>0.0 0.2 0.0 1.0</diffuse>
+        </material>
+      </visual>
+      <visual name="camera_lens_front_visual">
+        <pose>0.09 0 0.03 0 1.5707963267948966 0</pose>
+        <geometry>
+          <cylinder>
+            <length>0.011</length>
+            <radius>0.005</radius>
+          </cylinder>
+        </geometry>
+        <material>
+          <ambient>0.0 0.0 0.0 1.0</ambient>
+          <diffuse>0.0 0.0 0.0 1.0</diffuse>
+        </material>
+      </visual>
+
+
+      <!-- Physical robot has B0247 with LN008 with a 180 deg hor x 130 deg vert FOV
```

Comment:
Please adjust the modeling of the LN008 lens field of view to match the previously accepted standard of 2 cameras with 95 degree field for each camera.  In this case, the two cameras should be aligned with a +-45 offset to the current pose of the single camera.

Reply:
I believe the limits are currently set to 4m/S and 10min since validation is not complete.  I have updated per precedent and the cameras are offset +-45 with respect to the original camera.